rebasehelper/helpers/copr_helper.py

```diff
@@ -64,6 +65,15 @@ def create_project(cls, client, project, chroots, description, instructions, per
                                      instructions=instructions)
         except CoprRequestException as e:
             error = e.result.error
+            # In newer versions of copr, the response is JSON-like, not
+            # a simple text response but contains apostrophes instead of
+            # quotes, hence json.loads() is not usable. Read it as dict literal.
+            try:
+                response = ast.literal_eval(error)
+                if isinstance(response, dict):
+                    error = response['name'][0]
+            except (ValueError, IndexError, KeyError):
+                pass
```

Comment:
This is rather strange, and is bound to break eventually, but it's no worse than just checking for a substring I suppose.

Reply:
Agreed. The problem is that the `error` in this case looked something like `"{'name': "You already have project named 'rebase-helper-....'"}"` (escaping of quotes is left out as simplification). I initially wanted to replace `'` with `"` and parse it as json but due to the value, the replacement creates an invalid JSON so parsing fails. That's just odd formatting. Can you see a better way of handling this? I guess we could also ask around the copr team...

Reply:
Could it be a bug in the python binding? I wouldn't expect the service itself to return such a weird format

Reply:
> Could it be a bug in the python binding? I wouldn't expect the service itself to return such a weird format

I'm not sure, we can ask copr team for sure.

There is another copr-related problem I just found. rebase-helper creates a temporary project for an old build, but then it tries to create another project for a new build, but with the same name, and that fails. Should I open a separate issue for that?